docs/concepts/architecture.rst

```diff
@@ -4,62 +4,70 @@
 Flocker Cluster Architecture
 ============================
 
-This document describes the Flocker cluster's architecture.
-More accurately, it describes the architecture Flocker is moving towards, a transition that is still in progress.
+The Flocker cluster is comprised of the following sets of services:
 
-The Flocker cluster is composed of two sets of services:
+* :ref:`The control service <control-service>` exposes the :ref:`api`, with which you can manage and modify the configuration of your cluster.
+* :ref:`Flocker agents <flocker-agents>` are installed on each node in the cluster, and are used to modify the node to match the desired configuration of your cluster.
+* :ref:`The Flocker plugin for Docker <plugin>` is also installed on each node in your cluster if you want Flocker to manage your data volumes, while using other tools such as Docker, Docker Swarm, or Mesos to manage your containers.
 
-1. **The control service** that you can interact with using a :ref:`HTTP API<api>` to modify the desired configuration of the cluster.
-2. **Convergence agents** in charge of modifying the cluster state to match the desired configuration.
-   For example, if you're using Flocker's ZFS storage backend you will have ZFS-specific agents running on each node in the cluster.
+.. XXX FLOC-3598 add an architecture image here 
 
 .. _control-service:
 
-Control service
-===============
+The Control Service
+===================
 
-The control service is the integration point between:
+The control service is the brain of Flocker.
+It enables a user, or an automated orchestration framework, to monitor and modify the cluster state.
 
-* the human administrators or automated orchestration framework which determine the cluster configuration, and
-* the convergence agents that modify the cluster state accordingly.
+The control service accepts instructions either directly via the :ref:`api`, or via the Flocker CLI (which uses the API under the hood).
 
-The service consists of three components:
+When the control service has an instruction, it sends commands to the :ref:`Flocker agents <flocker-agents>`, and receives updates back.
```

Comment:
:grey_question: "has an instruction" is an unusual way to phrase it, but I understood the meaning. "receives" would be more standard, probably.